### PR TITLE
Add Toast notification on update image Wizard

### PR DESCRIPTION
**Pull request by adeliaferreira**
Need be tested to check if works
Signed-off-by: Adelia Ferreira <tay.figueira@gmail.com>

### CODE REVIEW

**Comment by boaz0** [comment]
@adeliaferreira run `npm run lint:js:fix` and `git add .` and `git commit -m "linting"`

File: src/Routes/ImageManager/UpdateImageWizard.js [hunk 1]
```diff
@@ -58,16 +63,15 @@ const UpdateImage = ({ navigateBack, updateImageID }) => {
       onSubmit={(values) => {
         const payload = {
           ...values,
+          username: data?.Installer?.Username,
```
Comment (boaz0) [inline src/Routes/ImageManager/UpdateImageWizard.js:66]: I think `username` and `credentials` should come from `values`.

**Comment by boaz0** [comment]
@matoval & @resoluteCoder can you review this?

**Comment by resoluteCoder** [comment]
@boaz0 Yes we were with @adeliaferreira when pushing code. After the removal of those two test lines 66/67. It should be good to merge.

**Review by resoluteCoder — APPROVED** [review]
Good to go once lines 66/67 are removed

**Comment by boaz0** [comment]
Thanks @resoluteCoder !

**Review by matoval — APPROVED** [review]
Looks good, once the fix is pushed.

**Pull request merged**